# SPDX-License-Identifier: GPL-2.0-only
# Preserve the reference GPIO-key built-in setting independently of meson-remote.
# Kconfig relocation only; full source-build verification is still required.
--- a/common_drivers/drivers/input/Kconfig
+++ b/common_drivers/drivers/input/Kconfig
@@ -9,6 +9,15 @@
 	  select m for ko
 	  select n for disable
 
+config AMLOGIC_GPIO_KEY
+	tristate "Amlogic gpio key device support"
+	default n
+	help
+	  Say Y here if
+	  you want to use
+	  the amlogic hold
+	  key.
+
 if AMLOGIC_INPUT
 source "$(COMMON_DRIVERS_DIR)/drivers/input/keyboard/Kconfig"
 source "$(COMMON_DRIVERS_DIR)/drivers/input/ir/Kconfig"
--- a/common_drivers/drivers/input/keyboard/Kconfig
+++ b/common_drivers/drivers/input/keyboard/Kconfig
@@ -1,13 +1,4 @@
 # SPDX-License-Identifier: (GPL-2.0+ OR MIT)
-
-config AMLOGIC_GPIO_KEY
-	tristate "Amlogic gpio key device support"
-	default n
-	help
-	  Say Y here if
-	  you want to use
-	  the amlogic hold
-	  key.
 
 config AMLOGIC_ADC_KEYPADS
 	bool "Meson ADC keypad support"
